source/common/upstream/strict_dns_cluster.cc

```diff
@@ -129,10 +129,12 @@ void StrictDnsClusterImpl::ResolveTarget::startResolve() {
             // make a new address that has port in it. We need to both support IPv6 as well as
             // potentially move port handling into the DNS interface itself, which would work better
             // for SRV.
+
             ASSERT(resp.address_ != nullptr);
+            Network::Address::InstanceConstSharedPtr new_address =
+                Network::Utility::getAddressWithPort(*(resp.address_), port_);
             new_hosts.emplace_back(new HostImpl(
-                parent_.info_, hostname_,
-                Network::Utility::getAddressWithPort(*(resp.address_), port_),
+                parent_.info_, hostname_, new_address,
```

Comment:
revert this file? I don't mind your change, but I think it's not really a functional change so you might as well make the PR smaller.

Reply:
Thanks, will revert.